packages/app-root/src/app/about/publications/page.js

```diff
@@ -11,7 +11,7 @@ import client from '../../../utils/contentfulClient.js'
 import mockResponse from './response.mock.json'
 
 export const metadata = {
-  title: 'Our Team',
+  title: 'Publications',
   description: 'The people who make the Zooniverse'
```

Comment:
This should change too, but I'm not sure to what?

Reply:
Good catch! I found a description in the older translation dictionary for app-content-pages, so I'll copy that over here.